packages/api/package.json

```diff
@@ -27,26 +27,21 @@
     "test:watch": "vitest --watch"
   },
   "dependencies": {
-    "@apollo/client": "3.10.4"
+    "@apollo/client": "3.11.1"
   },
   "devDependencies": {
     "@graphql-codegen/cli": "^5.0.2",
-    "@graphql-codegen/typescript-operations": "4.2.0",
+    "@graphql-codegen/typescript-operations": "4.2.3",
     "@graphql-codegen/typescript-react-apollo": "4.3.0",
     "@types/jest": "^29.5.12",
-    "@types/react": "^18.3.2",
+    "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
-    "@typescript-eslint/eslint-plugin": "^7.9.0",
-    "@typescript-eslint/parser": "^7.9.0",
-    "@vitejs/plugin-react": "^4.2.1",
-    "eslint": "^8.56.0",
-    "eslint-plugin-react-hooks": "^4.6.2",
-    "eslint-plugin-react-refresh": "^0.4.7",
-    "rimraf": "^5.0.7",
-    "typescript": "^5.4.5",
-    "vite": "^5.2.11",
+    "@vitejs/plugin-react": "^4.3.1",
+    "rimraf": "^6.0.1",
+    "typescript": "^5.5.4",
+    "vite": "^5.3.5",
     "vite-plugin-dts": "^3.9.1",
-    "vitest": "^1.6.0",
+    "vitest": "^2.0.4",
```

Comment:
Testen blijven netjes draaien?

Reply:
Ik zie geen verschil met de main branch, maar zie bij beide wel dezelfde warnings die de tests overigens niet laten falen. Misschien moeten we hier binnenkort even naar kijken.